adieu.py

```diff
@@ -0,0 +1,19 @@
+import inflect
+
+p = inflect.engine()
```

Comment:
Please rename variable "p" to something more meaningful.
Please read: https://1c-dn.com/library/rules_of_naming_variables/